TurkishFilter/sections/specific.txt

```diff
@@ -90,6 +90,8 @@ yenicaggazetesi.com.tr##.sidebar > div[class="space-3 flex center margin-bottom-
 yenicaggazetesi.com.tr##.container > div[class="space-2 flex center margin-bottom-md"]
 ||yenicaggazetesi.com.tr/d/other/cnt.jpg
 tebernuskirecci.com.tr###wrap > div[style="margin:0px 10% 25px 10%;min-height: 290px;"]
+koinmedya.com##.custom-html-widget > a[href^="https://www.binance.com/"][target="_blank"]
```

Comment:
too complicated
`a[href^="https://www.binance.com/tr/register?ref="]`

Reply:
And block banner by URL - 1.3 MB
||koinmedya.com/wp-content/uploads/*-300x600px-